core/idl/wallet/bitcoin/bitcoin_like_wallet.djinni

```diff
@@ -270,6 +270,12 @@ BitcoinLikeAccount = interface +c {
     broadcastRawTransaction(transaction: binary, callback: Callback<string>);
     broadcastTransaction(transaction: BitcoinLikeTransaction, callback: Callback<string>);
     buildTransaction(partial: optional<bool>): BitcoinLikeTransactionBuilder;
+    # Get fees from network, fees are ordered in descending order (i.e. fastest to slowest confirmation)
+    # Note: it would have been better to have this method on BitcoinLikeWallet
+    # but since BitcoinLikeWallet is not used anywhere, it's better to keep all
+    # specific methods under the same specific class so it will be easy to segratate
+    # when the right time comes !
+    getFees(callback: ListCallback<BigInt>);
```

Comment:
Will there be only ever 3 choices? Because today in live we map that to 3 wordings.

Reply:
Normally yes, explorers will always send 3 values, low, medium and high respectively 1, 3 and 6 confirmations (which is kind of a convention/standard)
By the way that's why I didn't a map with number of confirmations.
But let us know if this is important to you !
Thanks for the review 👍

Reply:
Not important